docs/source/index.md

```diff
@@ -0,0 +1,24 @@
+## Welcome to the NumS documentation!
+
+**NumS** is an open-source project developed at UC Berkeley RISE Lab.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+```{include} _includes/_discuss.md
+```
+
+## What documentation resource is right for me?
+
+```{include} _includes/_resources.md
+```
+
+## How can I get involved?
+
+NumS not only a [INSERT HERE], but also an active community of developers, researchers, and [INSERT HERE]
```

Comment:
Maybe try to resolve what to replace `INSERT HERE` with? It seems like an important part for visitors to get an impression of NumS :)

Reply:
I can't figure out what to write right now. I can fix it in soon in a future PR?